src/dodal/devices/aperturescatterguard.py

```diff
@@ -85,7 +84,7 @@ class ApertureScatterguard(StandardReadable):
     def __init__(self, prefix: str = "", name: str = "") -> None:
```

Comment:
`ApertureScatterguard` should inherit from both `StandardReadable` but also `Moveable`, which defines what the signature for `set()` should look like